sklearn/ensemble/gradient_boosting.py

```diff
@@ -375,7 +380,8 @@ def negative_gradient(self, y, pred, sample_weight=None, **kargs):
         if sample_weight is None:
             gamma = stats.scoreatpercentile(np.abs(diff), self.alpha * 100)
         else:
-            gamma = _weighted_percentile(np.abs(diff), sample_weight, self.alpha * 100)
+            gamma = _weighted_percentile(
```

Comment:
Why are you cleaning up flake8 issues for code that is not modified in this PR... It creates merged conflicts with other PRs. In general we try to enforce flake8 only for the code that is being modified in the PR... +1 for reverting these changes...